readme.md

```diff
@@ -760,6 +760,15 @@ Default: `Infinity`
 
 The maximum amount of items that should be emitted.
 
+###### pagination.requestLimit
+
+Type: `number`\
+Default: `10000`
+
+The maximum amount of request that should be triggered. [Retries on failure](#retry) are not count towards this limit.
+
+For example, if you need the first 10 pages but don't know how many items there are per page. It can also be helpful during development to avoid an infinite number of requests.
```

Comment:
> For example, if you need the first 10 pages but don't know how many items there are per page.

It's not clear how setting a request limit helps with this? It seems like setting a hard-coded limit goes against not knowing how many items there are.

Reply:
`are not count` -> `are not counted`

Reply:
@szmarczak it's fixed.